frontend/src/pages/HomePage.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import GreetingTitle from '../components/GreetingTitle.jsx';
+import GymSelector from '../components/GymSelector.jsx';
+import WorkoutCard from '../components/WorkoutCard.jsx';
+import gyms from '../data/mockedGyms';
+import '../global.css'; // make sure this is imported
+
+export default function HomePage() {
+  const [selectedGym, setSelectedGym] = useState(gyms[0].name);
+  const currentGym = gyms.find(g => g.name === selectedGym);
+
+  return (
+    <div style={{ padding: '20px' }}>
+      <div style={{ marginBottom: '30px' }}>
```

Comment:
instead to styling here, you can use a theme or use the CSS globals, this way you can use these styles to another pages because this should be action for all pages